src/plfa/Inference.lagda

```diff
@@ -506,8 +506,8 @@ _≟Tp_ : (A B : Type) → Dec (A ≡ B)
 ...  | yes refl | yes refl  =  yes refl
 \end{code}
 
-We will also need a couple of obvious lemmas: the domain
-and range of equal function types are equal.
+We will also need a couple of obvious lemmas; the domain
```

Comment:
I'm not so sure about this usage of a semicolon.

Reply:
or you can change to "We will also need a couple of obvious lemmas that the domain
and range of equal function types are equal."